multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -153,6 +159,38 @@
 logger = logging.getLogger(AUTOMM)
 
 
+@dataclass
+class ProblemProperty:
+    name: str  # Name of the problem
+    support_fit: bool = True  # Whether the problem type support `.fit()`
+    inference_ready: bool = False  # Support `.predict()` and `.evaluate()` without calling `.fit()`
+    is_matching: bool = False  # Whether the problem belongs to the matching category
+    experimental: bool = False  # Indicate whether the problem is experimental
+
+
+problem_property_dict = OrderedDict(
```

Comment:
@zhiqiangdon @bryanyzhu I added this problem_property_dict to record certain properties, like whether it supports `.fit()` or support `.predict()` without calling `.fit()`.

Reply:
Also @FANGAreNotGnu @cheungdaven

Reply:
Shall we later move this to constant?